sources/install/guacamole_install.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+# Author: The Exegol Project
+
+source common.sh
+
+set -e
+
+# Set variables
+GUACVERSION="1.5.1"
+SERVER="http://apache.org/dyn/closer.cgi?action=download&filename=guacamole/${GUACVERSION}"
+TOMCAT_HOME="/root/.tomcat"
+
+export DEBIAN_FRONTEND=noninteractive
+
+# Install dependencies
+apt install -y libcairo2-dev libjpeg62-turbo-dev libpng-dev libossp-uuid-dev libavcodec-dev libavformat-dev libavutil-dev \
+libswscale-dev freerdp2-dev libpango1.0-dev libssh2-1-dev libtelnet-dev libvncserver-dev libpulse-dev libssl-dev \
+libvorbis-dev libwebp-dev libwebsockets-dev freerdp2-x11 libtool-bin ghostscript dpkg-dev wget crudini libc-bin \
+tomcat9 tomcat9-admin tomcat9-common tomcat9-user kde-plasma-desktop xrdp xorgxrdp dbus-x11
+
+# Setup workspace
+mkdir /tmp/guacamole
+cd /tmp/guacamole/
+
+# Download requirements
+wget -q --show-progress -O guacamole-server-${GUACVERSION}.tar.gz ${SERVER}/source/guacamole-server-${GUACVERSION}.tar.gz
+tar -xzf guacamole-server-${GUACVERSION}.tar.gz
+
+mkdir -p /etc/guacamole/{extensions,lib}
+
+# Custom style
+mkdir custom-war
+cd custom-war
+wget -q --show-progress -O guacamole-${GUACVERSION}.war ${SERVER}/binary/guacamole-${GUACVERSION}.war
+unzip guacamole-${GUACVERSION}.war
+rm guacamole-${GUACVERSION}.war
+echo '<link rel="stylesheet" href="theme.css">' >> index.html
+cp /root/sources/assets/guacamole/theme.css ./
```

Comment:
Maybe create a src folder with the correct structure ? Might be a better way to edit the code / maintain it and also it could take only 1 cp command to deploy it as is.

Reply:
The problem is that if you want to update the version of Guacamole, it's easier to modify a variable (GUACVERSION) than to re-download manually, then modify the css to add it to the code.